packages/database/drizzle/0044_powerful_yellowjacket.sql

```diff
@@ -0,0 +1,418 @@
+	"score_term" text DEFAULT 'effective' NOT NULL,
+	"score_unit" text,
+	"score_max_value" numeric(40, 20),
+	"score_min_value" numeric(40, 20),
+	"is_binary" boolean DEFAULT false NOT NULL,
+	"completed_at" timestamp with time zone,
+	"metric_threshold" numeric,
+	"discussion_link" text,
+	"state" text DEFAULT 'draft' NOT NULL,
+	"summary" text
+);
+--> statement-breakpoint
+CREATE TABLE IF NOT EXISTS "v0_5_proposals" (
+	"proposal_addr" varchar(44) PRIMARY KEY NOT NULL,
+	"number" integer NOT NULL,
+	"proposer" varchar(44) NOT NULL,
+	"description_url" text NOT NULL,
+	"slot_enqueued" numeric NOT NULL,
+	"state" varchar NOT NULL,
+	"squads_proposal" varchar(44) NOT NULL,
+	"pass_amm_addr" varchar(44) NOT NULL,
+	"fail_amm_addr" varchar(44) NOT NULL,
+	"base_vault_addr" varchar(44) NOT NULL,
+	"quote_vault_addr" varchar(44) NOT NULL,
+	"dao_addr" varchar(44) NOT NULL,
+	"pass_lp_tokens_locked" numeric(20, 0) NOT NULL,
+	"fail_lp_tokens_locked" numeric(20, 0) NOT NULL,
+	"pda_bump" smallint NOT NULL,
+	"question_addr" varchar(44) NOT NULL,
+	"created_at" timestamp with time zone DEFAULT now() NOT NULL,
+	"finalized_at" timestamp with time zone,
+	"duration_in_slots" bigint DEFAULT 0 NOT NULL,
+	"updated_at_slot" bigint DEFAULT 0 NOT NULL
+);
+--> statement-breakpoint
+CREATE TABLE IF NOT EXISTS "v0_5_questions" (
+	"question_addr" varchar(44) PRIMARY KEY NOT NULL,
+	"is_resolved" boolean NOT NULL,
+	"oracle_addr" varchar(44) NOT NULL,
+	"num_outcomes" smallint NOT NULL,
+	"payout_numerators" jsonb NOT NULL,
+	"payout_denominator" bigint NOT NULL,
+	"question_id" jsonb NOT NULL,
+	"created_at" timestamp with time zone DEFAULT now() NOT NULL
+);
+--> statement-breakpoint
+CREATE TABLE IF NOT EXISTS "v0_5_refunds" (
+	"funding_record_addr" varchar(44) PRIMARY KEY NOT NULL,
+	"launch_addr" varchar(44) NOT NULL,
+	"funder_addr" varchar(44) NOT NULL,
+	"slot" bigint NOT NULL,
+	"timestamp" timestamp with time zone NOT NULL,
+	"quote_amount" numeric(20, 0) NOT NULL,
+	"created_at" timestamp with time zone DEFAULT now() NOT NULL
+);
+--> statement-breakpoint
+CREATE TABLE IF NOT EXISTS "v0_5_splits" (
+	"vault_addr" varchar(44) NOT NULL,
+	"vault_seq_num" bigint,
+	"signature" varchar(88) NOT NULL,
+	"slot" numeric NOT NULL,
+	"amount" bigint NOT NULL,
+	"created_at" timestamp with time zone DEFAULT now() NOT NULL
+);
+--> statement-breakpoint
+CREATE TABLE IF NOT EXISTS "v0_5_swaps" (
+	"id" bigserial PRIMARY KEY NOT NULL,
+	"signature" varchar(88) NOT NULL,
+	"slot" numeric NOT NULL,
+	"block_time" timestamp with time zone NOT NULL,
+	"swap_type" varchar NOT NULL,
+	"amm_addr" varchar(44) NOT NULL,
+	"user_addr" varchar(44) NOT NULL,
+	"amm_seq_num" bigint NOT NULL,
+	"input_amount" numeric NOT NULL,
+	"output_amount" numeric NOT NULL,
+	"created_at" timestamp with time zone DEFAULT now() NOT NULL
+);
+--> statement-breakpoint
+ALTER TABLE "v0_4_proposals" DROP CONSTRAINT "v0_4_proposals_pass_amm_addr_v0_4_amms_amm_addr_fk";
```

Comment:
These make sense, but you may need to comment them out to get the migration to run.